ibm/service/cdtektonpipeline/resource_ibm_cd_tekton_pipeline.go

```diff
@@ -119,11 +120,12 @@ func ResourceIBMTektonPipeline() *schema.Resource {
 						},
 						"service_instance_id": &schema.Schema{
```

Comment:
same as below a computed can't have inner attributes as Required/optional

Reply:
service_instance_id is only marked as `required` now, not computed

Reply:
I can update the description to make it clearer what this property is for

Reply:
Thanks for the clarification. This is a mistake, definitions should not be marked computed here. It appears to be an issue with [the generator](https://github.ibm.com/CloudEngineering/openapi-sdkgen) we use to create the provider.

I will look at updating the provider to remove erroneous Computed options such as the example you posted

Reply:
I think the issue here is that most of these properties are ones that *are* computed but only on the return object. They are not needed, nor should they be used, when making changes to a Tekton pipeline resource.

Would it make sense then to remove all of the return object properties and only include them in the data source for Tekton pipeline? That makes more sense to me

Reply:
We generally keep all computed properties in resource also else it makes us to dependent on one more datasouce if any one of properties need to referred in another resource.. 
May be we can work with generator team if a outer attribute is marked as computed then by default can we mark the inner attributes as computed irrespective
I think it will be as of readability issue for developers

Reply:
I'd like to hold off merging for the moment. The more I've worked on this and reviewed with my team, the more changes we'd like to make